Formula/lazygit.rb

```diff
@@ -23,12 +23,13 @@ def install
     system "go", "build", "-mod=vendor", *std_go_args(ldflags: ldflags)
   end
 
-  # lazygit is a terminal GUI, but it can be run in 'client mode' for example to write to git's todo file
+  # lazygit is a terminal GUI, but it can be run in 'client mode' to do certain tasks
   test do
     (testpath/"git-rebase-todo").write ""
-    ENV["LAZYGIT_DAEMON_KIND"] = "INTERACTIVE_REBASE"
-    ENV["LAZYGIT_REBASE_TODO"] = "foo"
+    ENV["LAZYGIT_DAEMON_KIND"] = "2" # cherry pick commit
+    ENV["LAZYGIT_DAEMON_INSTRUCTION"] = "{\"Todo\":\"pick 401a0c3\"}"
     system "#{bin}/lazygit", "git-rebase-todo"
-    assert_match "foo", (testpath/"git-rebase-todo").read
+    assert_match "pick 401a0c3", (testpath/"git-rebase-todo").read
```

Comment:
The original issue was that we changed the interface here so I'm now just updating to reflect that